README.md

```diff
@@ -74,15 +74,15 @@ If you configure any campaign parameter this configuration will overwrite defaul
 
 ```
 [MarketingCampaignsReporting]
-campaign_name = "pk_campaign,piwik_campaign,pk_cpn,utm_campaign"
-campaign_keyword = "pk_keyword,piwik_kwd,pk_kwd,utm_term"
+campaign_name = "pk_campaign,matomo_campaign,pk_cpn,utm_campaign"
+campaign_keyword = "pk_keyword,matomo_kwd,pk_kwd,utm_term"
```

Comment:
The core PR retains `piwik_campaign`/`piwik_kwd`, should this one do this too?

Reply:
Those are still included in the defaults of the plugin. But decided to already remove them from the README so new users directly use the new params only